sunpy/map/sources/suit.py

```diff
@@ -0,0 +1,103 @@
+"""
+SUIT Map sub class definitions
+"""
+import astropy.units as u
+from astropy.coordinates import CartesianRepresentation, HeliocentricMeanEcliptic
+from astropy.visualization import AsinhStretch, ImageNormalize
+
+from sunpy.map.mapbase import GenericMap
+from sunpy.map.sources.source_type import source_stretch
+
+__all__ = ["SUITMap"]
+__author__ = ["Rahul Gopalakrishnan"]
+__email__ = ["rahulg.astro@gmail.com"]
+
+class SUITMap(GenericMap):
+    """
+    SUIT Image Map.
+
+    The Solar Ultraviolet Imaging Telescope (SUIT) is one of the remote sensing
+    payloads on board the Aditya-L1 mission of the Indian Space Research Organization (ISRO)
+    that was launched on September 02, 2023. SUIT is designed to provide near-simultaneous full-disk
+    and region-of-interest images of the Sun at various heights, slicing through the photosphere
+    and chromosphere, employing an array of 11 scientifically calibrated filters (3 broad-band & 8 narrow-band)
+    strategically positioned within the wavelength range of 200 to 400 nanometers.
+    Located at the first Lagrange point, SUIT observes the Sun 24x7, without any interruption.
+
+    The SUIT data is available to download from the PRADAN interface of ISRO.
+
+    References
+    ----------
+    * `Home Page: <https://suit.iucaa.in/>`__
+    * `Mission Page: <https://suit.iucaa.in/about_SUIT>`__
+    * `Publications Page: <https://suit.iucaa.in/node/5>`__
+    * `Data Download PRADAN Page: <https://pradan.issdc.gov.in/al1>`__
+    """
+
+    def __init__(self, data, header, **kwargs):
+        super().__init__(data, header, **kwargs)
+        self._nickname = self.detector
+        filtername = header.get("FTR_NAME").strip()
```

Comment:
This will mean the defaulting later on works, without it the `.strip()` will fail.
```suggestion
        filtername = header.get("FTR_NAME", "").strip()
```